lang/types/value.go

```diff
@@ -203,6 +203,182 @@ func ValueOf(v reflect.Value) (Value, error) {
 	}
 }
 
+// Into mutates the given reflect.Value with the data represented by the Value
```

Comment:
Last nit: add a trailing period please. If you would like to be even more wordy and explicit that this modifies the reflect.Value even if this errors, then that would be welcome as well.

Reply:
Done